src/stylesheets/App.css

```diff
@@ -0,0 +1,288 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&family=Roboto+Slab:wght@100;200;300;400;500;600;700&display=swap');
+
```

Comment:
- [ ] Microverse really insists in this project to look exactly like the design provided in Zeplin, you did solve some issues but added new ones which I will point out and explain below:

- The zeplin design is this one:
![image](https://user-images.githubusercontent.com/38522578/133734615-45a8adbd-ba1e-4180-8df5-13842f7025fe.png)

- Your current design is this one:
![image](https://user-images.githubusercontent.com/38522578/133734353-b693f8ef-17a0-4cb8-b4be-7bdb8c4ee4c4.png)

- [ ] Please remove the underline in the navbar texts like bookstore cms and book, they are not present in the design given by Microverse.
- [ ] Please add the circular progress with a percentage that you had before, it was correct the last time that I checked this project.
- [ ] Please fix the spacing between the comment, remove and edit books options to comply with the project requirements.
- [ ] The right section where we can see the current chapter information is unaligned, please align it to comply with the project requirements.
- [ ] Please enlarge the bottom section where we are able to add a new book in order to match the width of the rest of the page elements as well as the dividing line that you currently have before that section.